client/components/Modal/PedalboardsModal.js

```diff
@@ -51,9 +35,9 @@ const PedalboardsModal = ({
         </Modal.Description>
       </Modal.Content>
     </Modal>
-  ) : null
+  )
 
-  return ModalOutput
+  return modalOpen && modalOutput
```

Comment:
Return early from functions. If you might not need the `modalOutput`, return first. Don't create elements that might not be used.